virtual-desktop/plugin-config/tslint.react.base.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["./tslint.base.linient.json", "tslint:latest", "tslint-react"]
```

Comment:
Also, need @1000TurquoisePogs 's input on whether or not these should go into the base JSON's for React and Angular. If any property in these files happens to be more restrictive than base, we risk an error to any existing app at compile time

I think these extends should go in "tslint.react.lenient.json" and "tslint.angular.lenient.json"

Reply:
The idea is to replace tslint.base.json with tslint.base.lenient.json in the future because it has some rules specific to angular that I am separating out. Also, the base file should only have the recommended rules and if some rules are not applicable to importer they can override it. So I have include only the base rules.